lib/serve/functions.js

```diff
@@ -104,6 +105,7 @@ function _startEmulator(options) {
   }).then(function(triggers) {
     var promises = _.map(triggers, function(trigger) {
       if (trigger.httpsTrigger) {
+        track('Functions Emulation (Trigger Type)', 'HTTPS');
```

Comment:
Yes, I'd like to do that once we support other function types. Right now, since we only support HTTPS triggers anyhow, this is the same as tracking whether the emulator was run.